model/src/test/kotlin/config/ExcludesTest.kt

```diff
@@ -92,22 +92,25 @@ class ExcludesTest : WordSpec() {
             "find the correct path excludes for a path" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
-                excludes.findPathExcludes("") should beEmpty()
-                excludes.findPathExcludes("path1") should containExactly(pathExclude1)
-                excludes.findPathExcludes("path2") should containExactly(pathExclude2)
-                excludes.findPathExcludes("test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/test.ext") should containExactly(pathExclude3)
-                excludes.findPathExcludes("directory/file.ext") should containExactly(pathExclude3, pathExclude4)
+                with(excludes) {
+                    findPathExcludes("") should beEmpty()
+                    findPathExcludes("path1") should containExactly(pathExclude1)
+                    findPathExcludes("path2") should containExactly(pathExclude2)
+                    findPathExcludes("test.ext") should containExactly(pathExclude3)
+                    findPathExcludes("directory/test.ext") should containExactly(pathExclude3)
+                    findPathExcludes("directory/file.ext") should containExactly(pathExclude3, pathExclude4)
+                }
             }
 
             "find the correct path excludes for a project" {
                 val excludes = Excludes(paths = listOf(pathExclude1, pathExclude2, pathExclude3, pathExclude4))
 
                 setProjects(project1, project2, project3)
-
-                excludes.findPathExcludes(project1, ortResult) should containExactly(pathExclude1)
-                excludes.findPathExcludes(project2, ortResult) should containExactly(pathExclude2)
-                excludes.findPathExcludes(project3, ortResult) should beEmpty()
+                with(excludes) {
```

Comment:
I'd prefer to keep the empty line above to separate arrange and assert.

Reply:
Done.